apps/condo/migrations/20221214152319-0201_auto_20221214_1223.js

```diff
@@ -0,0 +1,35 @@
+// auto generated by kmigrator
```

Comment:
is it possible to put this in one migration (200)?

Reply:
I made it specifically to make it easier to distinguish creating of a new table from editing existing one